test/facilitytype.test.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const should = require('chai').should();
+const server = require('../server/server');
+const request = require('supertest')(server);
+
+// describe('User', () => {
```

Comment:
@lytonnyemba Please work on this test... don't push branches with commented out code...